src/Functions/Public/relocation/Invoke-vRAOnboardingPlan.ps1

```diff
@@ -146,7 +146,7 @@
 "@
                             $ExecuteOnboardingPlan= Invoke-vRARestMethod -Method POST -URI $URI -Body $Body -Verbose:$VerbosePreference
 
-                            CalculateOutput
+                            CalculateOutput($ExecuteOnboardingPlan)
```

Comment:
Should the parameter be positional here and not in ()?

Reply:
I went through and removed the parenthesis where I found them. Should be in next commit.